electron{-source,-bin,-chromedriver}: 32.3.0 -> 32.3.1, 33.4.0 -> 33.4.1, 34.1.1 -> 34.2.0

emilylange opened the pull request:
- https://github.com/electron/electron/releases/tag/v32.3.1
- https://github.com/electron/electron/releases/tag/v33.4.1
- https://github.com/electron/electron/releases/tag/v34.2.0

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

emilylange committed:
electron_32-bin: 32.3.0 -> 32.3.1

- Changelog: https://github.com/electron/electron/releases/tag/v32.3.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v32.3.0...v32.3.1

emilylange committed:
electron_33-bin: 33.4.0 -> 33.4.1

- Changelog: https://github.com/electron/electron/releases/tag/v33.4.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v33.4.0...v33.4.1

emilylange committed:
electron_34-bin: 34.1.1 -> 34.2.0

- Changelog: https://github.com/electron/electron/releases/tag/v34.2.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v34.1.1...v34.2.0

emilylange committed:
electron-source.electron_32: 32.3.0 -> 32.3.1

- Changelog: https://github.com/electron/electron/releases/tag/v32.3.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v32.3.0...v32.3.1

emilylange committed:
electron-source.electron_33: 33.4.0 -> 33.4.1

- Changelog: https://github.com/electron/electron/releases/tag/v33.4.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v33.4.0...v33.4.1

emilylange committed:
electron-chromedriver_32: 32.3.0 -> 32.3.1

- Changelog: https://github.com/electron/electron/releases/tag/v32.3.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v32.3.0...v32.3.1

emilylange committed:
electron-chromedriver_33: 33.4.0 -> 33.4.1

- Changelog: https://github.com/electron/electron/releases/tag/v33.4.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v33.4.0...v33.4.1

emilylange committed:
electron-chromedriver_34: 34.1.1 -> 34.2.0

- Changelog: https://github.com/electron/electron/releases/tag/v34.2.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v34.1.1...v34.2.0

teutat3s reviewed (commented):
Thank you for taking care of this bump. The commit for the electron-source 34 version bump is currently missing.

emilylange committed:
electron_32: fix log spam when building on x86_64-linux as well

Follow-up to ce04dc5edc28423ea4cb3a63b13aa40a912e1b8f.

Turns out the very same log spam is happening on x86_64-linux and not
just on aarch64-linux but hydra.nixos.org does not enforce the log limit
of 64mb for some reason.

I should have figured.

emilylange commented:
> The commit for the electron-source 34 version bump is currently missing.

We don't build `electron_34` from source (yet?) :)

teutat3s commented:
Oops my bad. That explains it (:

emilylange commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 382361`

---
### `x86_64-linux`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>follow</li>
    <li>sharedown</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 97 packages built:</summary>
  <ul>
    <li>affine</li>
    <li>affine-bin</li>
    <li>antares</li>
    <li>antimatter-dimensions</li>
    <li>anytype</li>
    <li>appium-inspector</li>
    <li>bilibili</li>
    <li>bitwarden-desktop</li>
    <li>bitwarden-directory-connector</li>
    <li>blockbench</li>
    <li>bruno</li>
    <li>camunda-modeler</li>
    <li>caprine</li>
    <li>catalyst-browser</li>
    <li>chatd</li>
    <li>deltachat-desktop</li>
    <li>drawio</li>
    <li>drawio-headless</li>
    <li>electron (electron-bin, electron_34, electron_34-bin)</li>
    <li>electron-chromedriver (electron-chromedriver_34)</li>
    <li>electron-chromedriver_32</li>
    <li>electron-chromedriver_33</li>
    <li>electron_32</li>
    <li>electron_32-bin</li>
    <li>electron_33</li>
    <li>electron_33-bin</li>
    <li>element-desktop</li>
    <li>equibop</li>
    <li>fcast-receiver</li>
    <li>feishin</li>
    <li>freetube</li>
    <li>gdlauncher-carbon</li>
    <li>gfn-electron</li>
    <li>gitify</li>
    <li>goofcord</li>
    <li>google-chat-linux</li>
    <li>headset</li>
    <li>heroic</li>
    <li>heroic-unwrapped</li>
    <li>httptoolkit</li>
    <li>itch</li>
    <li>jitsi-meet-electron</li>
    <li>kando</li>
    <li>koodo-reader</li>
    <li>kuro</li>
    <li>legcord</li>
    <li>lx-music-desktop</li>
    <li>marktext</li>
    <li>mattermost-desktop</li>
    <li>morgen</li>
    <li>mqtt-explorer</li>
    <li>nix-tour</li>
    <li>obsidian</li>
    <li>open-stage-control</li>
    <li>openfortivpn-webview</li>
    <li>pandoc-drawio-filter</li>
    <li>pandoc-drawio-filter.dist</li>
    <li>penpot-desktop</li>
    <li>pocket-casts</li>
    <li>podman-desktop</li>
    <li>pritunl-client</li>
    <li>proton-pass</li>
    <li>protonmail-desktop</li>
    <li>python312Packages.mkdocs-drawio-exporter</li>
    <li>python312Packages.mkdocs-drawio-exporter.dist</li>
    <li>python313Packages.mkdocs-drawio-exporter</li>
    <li>python313Packages.mkdocs-drawio-exporter.dist</li>
    <li>r2modman</li>
    <li>redisinsight</li>
    <li>revolt-desktop</li>
    <li>ride</li>
    <li>sieve-editor-gui</li>
    <li>siyuan</li>
    <li>standardnotes</li>
    <li>stretchly</li>
    <li>super-productivity</li>
    <li>teams-for-linux</li>
    <li>terra-station</li>
    <li>tetrio-desktop</li>
    <li>thedesk</li>
    <li>threema-desktop</li>
    <li>trilium-next-desktop</li>
    <li>uhk-agent</li>
    <li>uhk-udev-rules</li>
    <li>uivonim</li>
    <li>vesktop</li>
    <li>vieb</li>
    <li>voicevox</li>
    <li>webcord</li>
    <li>webcord-vencord</li>
    <li>webtorrent_desktop</li>
    <li>yandex-music</li>
    <li>youtube-music</li>
    <li>ytdownloader</li>
    <li>ytmdesktop</li>
    <li>zepp-simulator</li>
    <li>zulip</li>
  </ul>
</details>

`follow` and `sharedown` failures are unrelated.

emilylange commented:
Backport fails due to 3f514d36a7c353152c22e72a36f1edd2f4b7c960.

Will stack the remaining commits onto #382813, given I intend to merge them together anyway.
